types/podcast/podcast-tests.ts

```diff
@@ -1,54 +1,67 @@
-import Podcast = require('podcast');
+import * as Podcast from 'podcast';
```

Comment:
This is incorrect. `Podcast` is a callable class and this import style is for namespaces which are not callable. Use 'import x = require('x')` as before instead.

Reply:
Thanks, done.